### PR TITLE
Cotco 634 json schema

### PR DESCRIPTION
**Pull request by greghmrc**

### CODE REVIEW

File: api/conf/1.0/schemas/transit_response.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      "type": "integer",
+      "title": "The Timeoutseconds Schema",
+      "default": 0,
+      "minimum": 0,
+      "examples": [
+        600
+      ]
+    }
+  }
+}
```
Comment (udinobi) [inline api/conf/1.0/schemas/transit_response.json]: What about adding the final newline ?

**Comment by udinobi** [comment]
It looks fine. However are you planning to also add the schema for the POST request ? That has a flat structure and does not include the `transitId` and the `timeoutSeconds` properties.

**Comment by greghmrc** [comment]
> It looks fine. However are you planning to also add the schema for the POST request ? That has a flat structure and does not include the `transitId` and the `timeoutSeconds` properties.

POST is for The App isn't it? Though, I can do it if you want.

**Comment by greghmrc** [comment]
All done.

**Comment by udinobi** [comment]
Awesome!

**Pull request merged**